containers/layouts/LayoutsSection.js

```diff
@@ -55,6 +55,9 @@ const LayoutsSection = () => {
     };
 
     const handleChangeViewMode = async (mode) => {
+        if (mode !== VIEW_MODE.GROUP) {
+            await api(updateStickyLabels(0));
```

Comment:
Use a constant please. If it doesn't exist, create it in `proton-shared/lib/constants`.